gobblin-compaction/src/main/java/gobblin/compaction/audit/KafkaAuditCountHttpClient.java

```diff
@@ -27,7 +26,6 @@
  * to perform audit count query.
  */
 @Slf4j
-@ThreadSafe
```

Comment:
Because this ThreadSafe is from http-core4.4.4, which seems not forward compatible. Change it to javax.annotation.ThreadSafe